modules/administration-guide/partials/proc_managing-backups-through-custom-resources.adoc

```diff
@@ -30,18 +30,24 @@ metadata:
 spec:
   backupServerConfigRef: backup-server-configuration <1>
 ----
-<1> Name of `CheBackupServerConfiguration` object that defines which backup server to use.
 
-To request a new backup create a new `CheClusterBackup` object.
-To reuse the same name, delete old object first.
+<1> Name of the `CheBackupServerConfiguration` object defining what backup server to use.
+
+* A request to create a new backup starts by creating the`CheClusterBackup` object.
+* Before reusing the same name for a new backup object, delete the old object first:
++
+[subs="+attributes,+quotes"]
+----
+{orch-cli} delete CheClusterBackup _<name>_ -n {prod-namespace}
+----
 
 [NOTE]
 ====
-Editing of `CheClusterBackup` objects has no effect.
+Editing the `CheClusterBackup` objects has no effect.
 ====
 
-To automatically setup and send a backup to internal backup server, instead of configuration reference specify `useInternalBackupServer` property:
-
+. Configure the automatic setup and sending of a backup to the internal backup server:
```

Comment:
This paragraph is not a continuation, but an alternative.
If a user doesn't want to configure backup server and ok to the internal backup server limitations, it is possible to request autoconfiguration from {prod-short} Operator by executing `{prod-cli} server:backup` command without arguments (note, in such case the preparation above should not be done)